tools/peptideshaker/fasta_cli.xml

```diff
@@ -0,0 +1,63 @@
+<tool id="fasta_cli" name="FastaCLI" version="1.0">
```

Comment:
@CarlosHorro can you please check out this tool: `DecoyDatabase Create decoy protein DB from forward protein DB. (Galaxy Version 2.3.0)` it for example installed on usegalaxy.eu. Does this tool do the same?

Reply:
Or this tool: Create Decoy Database (reverse) Creates a decoy search database by adding reverse sequences to an existing database

Reply:
I checked them but they don`t work properly with our tools as we require the headers to be set up in such a way that we can parse them properly in PeptideShaker, and the best way to ensure this is to use FastaCLI to add the decoys.

Reply:
That is unfortunate, but I guess nothing that we can avoid. Thanks for trying!

Reply:
Lets talk about FastaCLI once more. I was just using the SearchGUI version that is currently installed via the TS and realised that FastaCLI uses GBs of memory for a MBs size data base. I don't know if the problem persists in the latest version .. but it appears that the program is not the best to use. 

OpenMS includes the tool: "DecoyDatabase Create decoy protein DB from forward protein DB." here you can configure the header (append or prepend a configurable string). Would this be a good replacement?

Reply:
I'm afraid it isn't, as I said I tested it and it adds the string in different positions than the one we need in PS.

Reply:
playing the devel here ... can we adopt PS to support this other format?

Reply:
After talking with Harald he confirmed we need FastaCLI output at the moment, although we will try to extend that support in the future.
Also, we agreed FastaCLI cannot use GBs of memory for a MBs size database, it must be another different process. I've also done some tests with FastaCLI using uniprot-human-reviewed-trypsin-june-2017_concatenated_target_decoy.fasta and, as expected, it consumes less than 100 mb ram.